text/0007-weekly-inspo.md

```diff
@@ -0,0 +1,301 @@
+creatively designed to show off our inspo in an unique fashion.
+
+Ultimately, the purpose of `xplr` is to constantly expose our dev team to the greater 
+world of dev and to familarize everyone with documentation, the PR process, and 
+exploration.
+
+## Why are we doing this?
+
+One of the things that we really stress at Creative Labs is having taste and 
+inspiration. Inspo is a big part of creativity and allows us to create better work. 
+Although inspo is usually something that is associated with the design side, exposing 
+ourselves to different dev concepts would make us better developers and empower us to 
+challenge our designers to make even more creative work.
+
+## What is the high level design/implementation?
+
+`xplr` can be built in an entirely self-contained environment. `xplr` will be built
+as a `Next.js` application that will be deployed to Netlify. There isn't a need for
+any servers as `Next.js` comes with an `api` that Netlify will deploy as a AWS function
+for us.
+
+### Organization
+
+The `xplr` repository will have a standard `Next.js` structure. On top of the standard
+configuration, we will also need to have a `_posts` and `_team` directory to keep 
+track of each inspo post and each team member. See [Appendix](#organization-(expanded))
+for more details on organization.
+
+### Verification
+
+In order to reduce the chance of introducing bugs into our application. We will need
+to write some custom linters to test the efficacy of the our posts.
+
+**Posts Linter**
+
+* Check that the author exists in the team
+* `title` is identical to the title of the article
+* `slug` is identical to file name
+* Check that `tag` is delimited white spaces and is an available tag in the'
+  `_posts/tags.yml` file
+
+**Team Members Linter**
+
+* Check if `name` is capitalized
+* Check that the file name is the `name` but in `kebab-case`
+
+**Markdown Linter**
+
+* Make sure all markdown files are only 80 characters wide
+* Warnings for spelling would be a nice to have
+  * For a spell check GH Action: https://github.com/rojopolis/spellcheck-github-actions
+* Hopefully we can find a linter out there for this already
+
+### Search
+
+We can accomplish search using `Next.js` built in API system. Here is an [example](https://medium.com/@matswainson/building-a-search-component-for-your-next-js-markdown-blog-9e75e0e7d210https://medium.com/@matswainson/building-a-search-component-for-your-next-js-markdown-blog-9e75e0e7d210)
+search system using `Next.js`.
+
+### Change Log
+
+It would be nice to keep track of a `CHANGE_LOG` to see the week by week updates that
+happen to the respository. The functionality for this is mostly internal, and can serve
+as a method to expose newer board members to industry practices! We would most likely 
+have a `GitHub` action that runs at the time of dev team meeting each week, that would
+automatically create a pull request to update the `CHANGE_LOG`.
```

Comment:
what exactly would the changelog contain? if it's an automated thing, then how is it different from just a list of commits from the past week? i ask since you say this is primarily for internal purposes.

on a similar note, wouldn't it be easier and accomplish the same effect to just have PR titles/bodies to have a standard structure and amount of information? so that when they're squashed into main, the commit messages are detailed enough to serve as a changelog in themselves

Reply:
Hmmm now that I think about it.. ur right.. why do we need a Change Log???

Originally I think I wanted a way to organize our posts, site, and api updates in a weekly place to give our team a taste of what it's like for managing a package/api 

But I think ur right.. the function doesn't match

Reply:
yeah, i think a different project (like an actual application) would be more suitable for a changelog. maybe for RFC #2 perhaps :eyes:

Reply:
Ooo I like that idea